tools/utils/runPostShiboken.sh

```diff
@@ -47,7 +47,7 @@ GCC_DIAG_OFF(uninitialized)\
 GCC_DIAG_UNUSED_LOCAL_TYPEDEFS_OFF\
 #include <shiboken.h> // produces many warnings@' -i'.bak' Engine/NatronEngine/*.cpp Gui/NatronGui/*.cpp
 
-sed -e 's@// Extra includes@// Extra includes\
```

Comment:
rather than changing the command so that it breaks shiboken1, why not do the two commands, so that it works both with shiboken1 and shiboken2?

Reply:
Your way is smarter with what was proposed, gonna proceed by doing what you've described.